src/ORM/Table.php

```diff
@@ -860,6 +860,38 @@ public function addBehavior($name, array $options = [])
         return $this;
     }
 
+    /**
+     * Add Many Behaviors
+     *
+     * Adds an array of behaviors to the table's behavior collection.
+     *
+     * Example:
+     *
+     * ```
+     * $this->addBehaviors([
+     *      'Timestamp',
+     *      'Tree' => ['level' => 'level'],
+     * ]);
+     * ```
+     *
+     * @param array $behaviors All of the behaviors to load.
+     * @return $this
+     * @throws \RuntimeException If a behavior is being reloaded.
+     */
+    public function addBehaviors(array $behaviors)
+    {
+        foreach ($behaviors as $name => $options) {
+            if (is_int($name) === true) {
```

Comment:
`if (is_int(...))` could suffice (the return value is 100% bool always).

Reply:
Affirm. I'll update. It's just habit. I try to use strict comparison as much as possible.